cdp_scrapers/instances/seattle-static.json

```diff
@@ -0,0 +1,166 @@
+{
+    "persons": {
+        "Alex Pedersen": {
+            "name": "Alex Pedersen",
+            "is_active": true,
+            "router_string": null,
+            "email": null,
+            "phone": null,
+            "website": null,
+            "picture_uri": "http://www.seattle.gov//assets/images/Council/Members/Pedersen/Councilmember-Alex-Pedersen_homepage-banner.jpg",
+            "seat": {
+                "name": "Position 4",
+                "electoral_area": "District 4",
+                "electoral_type": null,
+                "image_uri": "https://www.seattle.gov/Images/Clerk/DistrictsMap.jpg",
```

Comment:
Since we don't have district maps made for each of these, can we just get images of like landmarks from each district? I can help find some as well but having the same image for each seat is a bit odd imo.

Reply:
We could also make our own district maps. But for now... easier to just find an image of a landmark in each district.

Reply:
> Since we don't have district maps made for each of these, can we just get images of like landmarks from each district? I can help find some as well but having the same image for each seat is a bit odd imo.

Good idea.

Reply:
Thoughts on pictures used in the "Explore More Seattle Neighborhoods" section near the bottom of this web page? https://visitseattle.org/neighborhoods/belltown/

Reply:
Yea those are great! Nice find!